tests/features/bootstrap/FeatureContext.php

```diff
@@ -386,6 +386,42 @@ public function assertFieldOptionSelected($option, $select) {
   }
 
   /**
+   * Checks that a certain radio input is checked in a specific field.
+   *
+   * @param string $radio
+   *   The label of the radio input to find.
+   * @param string $field
+   *   The label of the field the radio is part of.
+   *
+   * @throws \Exception
+   *   Thrown when the field or the radio is not found, or if the radio is not
+   *   checked.
+   *
+   * @Then the radio :radio from field :field should be checked
+   */
+  public function assertFieldRadioSelected($radio, $field) {
```

Comment:
No just rename the step definition to "selected" like I proposed. One does not check a radio button.

* Checking is for checkboxes
* Selecting is for radio buttons